pallets/common/src/erc.rs

```diff
@@ -29,29 +31,39 @@ use alloc::format;
 
 use crate::{Pallet, CollectionHandle, Config, CollectionProperties};
 
+/// Events for etherium collection helper.
```

Comment:
Still etherium :)

There are a couple more of these typos, please search for `etherium` in the repo root and adjust all instances.